src/components/ResourceCard/ResourceCard.js

```diff
@@ -118,10 +125,12 @@ CardHeader.propTypes = {
   resource: shape({}).isRequired,
   collectionId: string.isRequired,
```

Comment:
I'm seeing a proptype warning that `CardHeader` is not getting the required `collectionId`